db/migrate/20240123104811_change_fees_grouped_by_type.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class ChangeFeesGroupedByType < ActiveRecord::Migration[7.0]
+  def change
+    remove_column :fees, :grouped_by, :string, array: true, default: []
+    add_column :fees, :grouped_by, :jsonb, null: false, default: {}
+
+    remove_column :cached_aggregations, :grouped_by, :string, array: true, default: []
```

Comment:
Rails/BulkChangeTable: You can use `change_table :cached_aggregations, bulk: true` to combine alter queries. (https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-change_table, https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/Table.html)